tools/fuchsia/copy_debug_symbols.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+#
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+""" Gather the build_id, prefix_dir, and exec_name given the path to executable
+    also copies to the specified destination.
+"""
+
+import argparse
+import json
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+
+def Touch(fname):
+  with open(fname, 'a'):
+    os.utime(fname, None)
+
+
+def GetBuildIdParts(exec_path):
+  file_out = subprocess.check_output(['file', exec_path])
```

Comment:
Dang, is there no other way to get the build ID for an ELF binary?

Reply:
This doesn't seem to work on cirrus for some reason. I'm wondering if `file` is sandboxed away from us. I will look for alternatives.

Reply:
The Fuchsia toolchain includes a version of `llvm-readelf`.

Try extracting the build ID with:
`llvm-readelf --hex-dump=.note.gnu.build-id [executable]`

Reply:
Great idea, I switched to using this,